script.js

```diff
@@ -0,0 +1,18 @@
+function removeTransition(e) {
+  console.log(e.propertyName);
+  if (e.propertyName !== 'transform') return;
+  e.target.classList.remove('playing');
+}
+
+function playSound(e) {
+  const audio = document.querySelector(`audio[data-key="${e.keyCode}"]`);
+  const key = document.querySelector(`div[data-key="${e.keyCode}"]`);
+  if (!audio) return;
```

Comment:
There is no need to select `key`  and create a constant for it if there is no audio. So, consider change the order of 9th and 10th lines here.

Reply:
DONE!